provisioning/roles/php-xdebug/tasks/main.yml

```diff
@@ -5,6 +5,7 @@
 - include_vars: "{{ item }}"
```

Comment:
mainly for the xdebug path in php.ini (it's different in different php versions). Or do you mean something else?